crates/configuration/src/shared/types.rs

```diff
@@ -0,0 +1,93 @@
+#[derive(Debug, Clone)]
+pub struct MultiAddress(String);
+
+#[derive(Debug, Clone)]
+pub struct IpAddress(String);
+
+#[derive(Debug, Clone)]
+pub struct Timeout(String);
+
+#[derive(Debug, Clone)]
+pub struct Command(String);
```

Comment:
We have a couple of options to use these `new-type` in the `configuration`, we can change the field visibility (e.g `pub(crate)` ) or we can add a constructor (e.g `pub fn new(cmd: impl Into<String>`))

I think the later is more appropriated but I'm open to discuss other ideas.